etc/updateCopyRightYear.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-#Updates the year in our copyright statement. i.e. "* (C) Copyright 2003-2016," gets replaced by "* (C) Copyright 2003-[current_year],"
+#Updates the year in our copyright statement. i.e. "* (C) Copyright 2003-2018," gets replaced by "* (C) Copyright 2003-[current_year],"
```

Comment:
There's no need to update this since the example is showing an out-of-date year intentionally.

Reply:
ok sir.

Reply:
Sir I have sent you an email mentioning solution of gsoc-warmup. Please check that.